changelog/27510.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+agent: Add the ability to dump pprof to the filesystem using SIGUSR2
+```
```

Comment:
```suggestion
```release-note:improvement
agent: Add the ability to dump pprof to the filesystem using SIGUSR2
```

```release-note:improvement
proxy: Add the ability to dump pprof to the filesystem using SIGUSR2
```
```

Reply:
okay the suggestion got a little messed up with the triple backticks but we should essentially have two of these, one for Agent, one for Proxy

They can be identical other than the component